.travis.yml

```diff
@@ -5,7 +5,8 @@ node_js:
   - '0.12'
   - '4'
 
-sudo: false
+dist: trusty
+sudo: true
```

Comment:
@vladikoff looks like it needs to have `sudo: true` or else it fails with https://travis-ci.org/mozilla/fxa-oauth-server/jobs/118151093